### PR TITLE
add torch.softmax

### PR DESCRIPTION
**Pull request by zh-hike**
add torch.softmax doc

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.softmax.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+torch.softmax(input, dim, *, dtype=None)
+```
+
+### [paddle.nn.functional.softmax](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/functional/softmax_cn.html)
+
+```python
+paddle.nn.functional.softmax(x, axis=-1, dtype=None, name=None)
+```
+
+仅参数名不一致
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.softmax.md]: 这个属于差异简介部分，可以多写一点文字；参考下其他的

Comment (zh-hike) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.softmax.md]: done
